include/constants.h

```diff
@@ -23,6 +23,38 @@
 #ifndef CONSTANTS_H
 #define CONSTANTS_H
 
+#if defined(__GNUC__) || defined(__clang__)
+#if __has_include("user_constants.h")
+#include "user_constants.h"
+#endif
+#endif
+
+/*
+ * Hypervisor Version
+ *
+ * Uses http://semver.org
+ */
+#define BAREFLANK_VERSION_MAJOR 1ULL
+#define BAREFLANK_VERSION_MINOR 1ULL
```

Comment:
Might want to be careful setting the version before the tag. This type of versioning can cause problems in the future.